src/lib.rs

```diff
@@ -517,6 +594,16 @@ mod tests {
         assert_eq!(x.find_gte(257), None);
     }
 
+    #[test]
+    fn check_into_iter() {
```

Comment:
Can you also add a test for `.iter`?

Reply:
Just to make sure. There is already a doc-test on `OC::iter()`. Do you want to have additional explicit test?

Reply:
Yeah, always handy to have an actual unit test in addition to the doc tests. For example, doc tests currently (unless something has changed recently) don't get counted towards coverage. Also, doc tests tend to focus on "how does this API work" rather than "let's push on some of the corner cases". For instance, would be good to test that iteration also works correctly on an empty collection.